plugins/admin/pkg/command/registry/remove.go

```diff
@@ -0,0 +1,148 @@
+// Copyright © 2020 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package registry
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/client-contrib/plugins/admin/pkg"
+
+	"github.com/spf13/cobra"
+)
+
+var username string
+var server string
+
+// NewPrRmCommand represents the remove command
+func NewPrRmCommand(p *pkg.AdminParams) *cobra.Command {
+	var prRmCmd = &cobra.Command{
+		Use:   "remove",
+		Short: "Remove registry with credentials",
+		Long: `Remove registry with credentials that used for Service deployment
+For example:
```

Comment:
Can we use the **cobra.Command.Example** field to set command examples? As I described in #43 , it will give us a better usage output. @zhanggbj How do you think about it? If we agree to use **Example** field. I think @12345lcr can adopt it in his PR first.  
BTW, if we change to use **Example** field, we'd better to indent example lines with two spaces, that will make usage look better.

Reply:
@chaozbj
Thanks! I agree with that cobra.Command.Example is a formal way to do it and also it's better to follow `kn` client conventions to keep a similar user experience.

Reply:
Great findings. Fixed.